scripts/gz10/gz10.py

```diff
@@ -150,9 +148,7 @@ def _generate_examples(self, files, object_ids=None):
 
                     example = {
                         "gz10_label": data["ans"][i].astype(np.int32),
-                        "dec": data["dec"][i].astype(np.float32),
-                        "ra": data["ra"][i].astype(np.float32),
```

Comment:
Why are we dropping these keys? I think they should be included for cross matching.

Reply:
no, cross matching is not done at that level